pypeit/data/utils.py

```diff
@@ -225,6 +229,51 @@ def get_skisim_filepath(skisim_file):
     # Return the path to the `skisim` file
     return skisim_path
 
+def get_spectrograph_gain_ronoise_dirpath(spectrograph_name):
```

Comment:
Because the files in `pypeit/data/spectrograph` are not overly large, we don't need to resort to only keeping them on GitHub.  As such, this new method is not needed.

Reply:
> Do we want to have it fail to a particular value if the HDU is munged? Like the current values or most recently measured values in the file?
@bpholden if the hdu is not available, we'll use the value that we were using before. For the `MJD` header keyword, we have already a system in place for which if `MJD` does not exist, we'll use the observation date and convert it to mjd. I think that should secure us that this code will not crash.

Reply:
Removed!